internal/simulator/datastore_create.go

```diff
@@ -0,0 +1,85 @@
+// (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+
+package simulator
+
+import (
+	_ "embed"
+
+	"github.com/h2non/gock"
+)
+
+//go:embed fixtures/datastores/create/async1.json
+var dsAsync1 string
+
+//go:embed fixtures/datastores/create/async2.json
+var dsAsync2 string
+
+//go:embed fixtures/datastores/create/async3.json
+var dsAsync3 string
+
+//go:embed fixtures/datastores/create/async4.json
+var dsAsync4 string
+
+//go:embed fixtures/datastores/create/async5.json
+var dsAsync5 string
+
+//go:embed fixtures/datastores/create/filter.json
+var dsFilter string
+
+//go:embed fixtures/datastores/create/get.json
+var dsDet string
```

Comment:
`dsDet` instead of `dsGet` typo?

Reply:
Yes, thanks. Curious that this was consistent on line 84.